app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```

Comment:
- [ ] you should add an index and show actions so it won't produce an error when visiting their page